torch/_ops.py

```diff
@@ -32,31 +32,30 @@ def __init__(self, overloadpacket, op, schema):
         self._op = op
         self._schema = schema
         self._overloadpacket = overloadpacket
+        self.__name__ = 'default' if schema.overload_name == '' else schema.overload_name
 
     # it's a no-op since OpOverload object is immutable and must be unique for a given op overload.
     def __deepcopy__(self, memo=None):
         return self
 
-    def __str__(self):
-        return "OpOverload(op='{}.{}', overload='{}')".format(*self._schema.name.split("::"), self.overload_name)
+    def __repr__(self):
+        return "<OpOverload(op='{}.{}', overload='{}')>".format(*self._schema.name.split("::"), self.__name__)
```

Comment:
Why the `<` `>` ?

Reply:
see https://github.com/pytorch/pytorch/pull/72673#discussion_r819129798

Reply:
Makes sense